host/integration_test.go

```diff
@@ -79,3 +81,15 @@ func (s *integrationSuite) decodePayloadsString(ps *commonpb.Payloads) (r string
 	s.NoError(payloads.Decode(ps, &r))
 	return
 }
+
+func (s *integrationSuite) decodePayloadsInt(ps *commonpb.Payloads) (r int) {
```

Comment:
this is not used anywhere in this PR?

Reply:
Oops, I had the uses in another commit, it must have gotten lost as I was shuffling and rebasing.